src/OrchardCore.Modules/OrchardCore.ContentLocalization/DefaultContentLocalizationManager.cs

```diff
@@ -100,5 +108,77 @@ public async Task<ContentItem> LocalizeAsync(ContentItem content, string targetC
             _session.Save(cloned);
             return cloned;
         }
+
+        public async Task<IDictionary<string, ContentItem>> DeduplicateContentItemsAsync(IEnumerable<ContentItem> contentItems)
```

Comment:
Maybe I misunderstood the naming convention of Async. 

Should a method name end with Async when they don't await the last statement ? @jtkech

Reply:
It's just a different way to write your method body, as soon as your method is seen by the caller as an awaitable `Task` the convention is applied.

Yes, if in all your code paths you don't need to await others async methods (as in a loop or to init a var to be used later), you don't have to use the `async` keyword and all your return paths can directly return a `Task` (e.g another async method, a `Task.FromResult` in another place ...) without awaiting it (preventing some more allocations).

But as said it's just a different way, if the last statement is awaited the convention is still applied.